Dart/testData/dart_style/whitespace/cascades.stmt

```diff
@@ -15,6 +15,10 @@ list
   ..add("bar");
 <<<
 list..add("baz")..add("bar");
+>>> multiple cascades get the same line when the method names are the same
```

Comment:
This change makes test data different from what is there in dart_style original repo. It makes it difficult to port new changes from that repo.
I suggest to keep intellij-plugins/Dart/testData/dart_style 100% equivalent to dart_style/test and to update tests of DartFormatterTest instead.

Also as a rule all fixes are committed with working tests. So if we keep DartStyleTest disabled we need to add the case to DartFormatterTest (or update any its existing test data).
Other option is to enable DartStyleTest, but only those cases that pass. So with each next fix we'll enable more and more cases and some lucky day the test will fully pass.